Tools/gen_manifest

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import subprocess
+import re
+from shutil import copyfile
+runtime_libs = []
+def init_runtime_libs () :
+  runtime_libs.append('libc')
```

Comment:
IMHO it would be clearer and simpler to just list all this stuff inline in the `runtime_libs` initialization.

Reply:
Thanks for suggestion. DONE.